lib/elixir/lib/kernel.ex

```diff
@@ -525,15 +526,16 @@ defmodule Kernel do
       :b
 
   """
-  @spec max(first, second) :: (first | second) when first: term, second: term
-  def max(first, second) do
-    :erlang.max(first, second)
+  @spec max(left, right) :: (left | right) when left: term, right: term
+  def max(left, right) do
+    :erlang.max(left, right)
```

Comment:
I was having this discussion with @lexmag and I am convinced `first` and `second` don't make sense here in particular over other operators, since order matters in most operators (but still we call things `left` and `right`). I think we can settle for `value1` and `value2`?